openpower/package/barreleye-xml/barreleye-xml.mk

```diff
@@ -40,9 +40,11 @@ define BARRELEYE_XML_BUILD_CMDS
             $(MRW_HB_TOOLS)/attribute_types.xml \
             $(MRW_HB_TOOLS)/attribute_types_hb.xml \
             $(MRW_HB_TOOLS)/attribute_types_openpower.xml \
+            $(MRW_HB_TOOLS)/attribute_types_oppowervm.xml \
```

Comment:
Should we change these to simply be wildcards, so we don't have to keep updating them?

$(wildcard $(MRW_HB_TOOLS)/attribute_types*.xml) \
$(wildcard $(MRW_HB_TOOLS)/target_types*.xml)